src/data_traits.rs

```diff
@@ -96,6 +96,15 @@ pub unsafe trait Data: RawData {
     where
         Self::Elem: Clone,
         D: Dimension;
+
+    fn to_share<D>(self_: &ArrayBase<Self, D>) -> ArrayBase<OwnedArcRepr<Self::Elem>, D>
```

Comment:
Very nice, let's name this method `to_shared` IMO

Reply:
Okay, they should be consistent